app/templates/people/index.hbs

```diff
@@ -112,17 +112,7 @@
         {{#if this.hasErrored}}
           <TableMessage::Error />
         {{else if this.hasNoResults}}
-          <TableMessage>
-            <p>
-              Er werden geen zoekresultaten gevonden. Kijk na op spelfouten, of
-              probeer een andere zoekopdracht. Indien een persoon ontbreekt,
-              meldt dit aan
-              <a
-                href="mailto:{{config 'contactEmail'}}"
-                class="au-c-link"
-              >{{config "contactEmail"}}</a>.
-            </p>
-          </TableMessage>
+          <TableMessage::NoResult />
```

Comment:
Similar remark, I don't see this change in the designs and I'm afraid that doing this change will encourage the users to create persons that might already exist but who just don't have a position attached to them yet, which is the opposite of what we are trying to achieve with this ticket no ?